### PR TITLE
add anon mining miner

### PR DESCRIPTION
**Pull request by kwhuo68**
**Summary of changes**
Changes introduced in this pull request:
-Anonymity mining Miner contract functionality

### CODE REVIEW

**Comment by drewstone** [comment]
@kwhuo68 what's the status of this?

**Comment by kwhuo68** [comment]
> @kwhuo68 what's the status of this?

Paused this - working on https://github.com/webb-tools/protocol-solidity/pull/225 which should be ready for review soon

File: pallets/anonymity-mining/src/lib.rs [hunk 1]
```diff
@@ -133,6 +143,38 @@ pub mod pallet {
...
+	pub type TreeUpdateVerifier<T: Config<I>, I: 'static = ()> =
+		StorageValue<_, T::AccountId, ValueQuery>;
+
+	#[pallet::storage]
+	#[pallet::getter(fn get_governance)]
+	pub type Governance<T: Config<I>, I: 'static = ()> = StorageValue<_, T::AccountId, ValueQuery>;
+
+	#[pallet::storage]
+	#[pallet::getter(fn get_tornado_trees)]
+	pub type TornadoTrees<T: Config<I>, I: 'static = ()> =
```
Comment (drewstone) [inline pallets/anonymity-mining/src/lib.rs:174]:
```suggestion
	pub type RewardTrackingTrees<T: Config<I>, I: 'static = ()> =
```

**Review by drewstone — CHANGES_REQUESTED** [review]
What's the status of this @kwhuo68 ?

**Pull request closed without merging**